src/main/java/de/tum/in/www1/artemis/web/rest/LectureResource.java

```diff
@@ -71,14 +77,14 @@ public LectureResource(LectureRepository lectureRepository, LectureService lectu
      * @throws URISyntaxException if the Location URI syntax is incorrect
      */
     @PostMapping("/lectures")
-    @PreAuthorize("hasRole('INSTRUCTOR')")
+    @PreAuthorize("hasRole('EDITOR')")
     public ResponseEntity<Lecture> createLecture(@RequestBody Lecture lecture) throws URISyntaxException {
         log.debug("REST request to save Lecture : {}", lecture);
         if (lecture.getId() != null) {
             throw new BadRequestAlertException("A new lecture cannot already have an ID", ENTITY_NAME, "idexists");
         }
         User user = userRepository.getUserWithGroupsAndAuthorities();
-        if (!authCheckService.isInstructorInCourse(lecture.getCourse(), user) && !authCheckService.isAdmin(user)) {
+        if (!(authCheckService.isEditorInCourse(lecture.getCourse(), user) || authCheckService.isInstructorInCourse(lecture.getCourse(), user) || authCheckService.isAdmin(user))) {
```

Comment:
```suggestion
        if (!authCheckService.isAtLeastEditorInCourse(lecture.getCourse(), user)) {
```
I think we could get rid of these separate authorization checks and simply use the frequently used `isAtLeastXXX` check, which shortens the if clause and handles superior roles as well :)

Reply:
Good idea, thanks!